plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Parser.hs

```diff
@@ -0,0 +1,103 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE RankNTypes          #-}
```

Comment:
Or this?

Reply:
Similarly in the PIR and PLC parsers.

Reply:
`OverloadedStrings` is still needed but I removed the others.